CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Prerelease
+
+### Fixes
+- **[BREAKING]**: `IEntriesClient`
+  - Fix `GetDocumentContentTypeAsync` return type from `Task` to `Task<HttpResponseHead>` to allow retrieving response headers.
```

Comment:
is this a breaking change?
The only way a client can have used current version, is to have called the method and assigned the return value to a variable.
If **f** is a function with return type Task, and we have this statement, 
Task t = f();
then if we change f's return type to Task<Boolean>, this statement is still valid.

Reply:
ok i see. i removed the breaking text